modules/serverless-config-minimal-workloads-serving.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+//
+// * /knative-serving/scalability-and-performance-serving.adoc
+
+
+:_mod-docs-content-type: PROCEDURE
+[id="serverless-config-minimal-workloads-serving_{context}"]
+= Configuring Serving for minimal workloads
+
+.Procedure
+
+* You can configure Knative Serving for minimal workloads using the `KnativeServing` Custom Resource (CR):
++
+.A minimal workload configuration in KnativeServing CR
+[source,yaml]
+----
+apiVersion: operator.knative.dev/v1beta1
+kind: KnativeServing
+metadata:
+  name: knative-serving
+  namespace: knative-serving
+spec:
+  high-availability:
+    replicas: 1 <1>
+  workloads:
+    - name: activator
+      replicas: 2 <2>
+      resources:
+        - container: activator
+          requests:
+            cpu: 250m <3>
+            memory: 60Mi <4>
+          limits:
+            cpu: 1000m
+            memory: 600Mi
+    - name: controller
+      replicas: 1 <6>
+      resources:
+        - container: controller
+          requests:
+            cpu: 10m
+            memory: 100Mi <4>
+          limits: <5>
+            cpu: 200m
+            memory: 300Mi
+    - name: webhook
+      replicas: 2 <6>
+      resources:
+        - container: webhook
+          requests:
+            cpu: 100m <7>
+            memory: 60Mi <4>
+          limits:
+            cpu: 200m
+            memory: 200Mi
+  podDisruptionBudgets: <8>
+    - name: activator-pdb
+      minAvailable: 1
+    - name: webhook-pdb
+      minAvailable: 1
+----
+<1> Setting this to `1` scales all system components to one replica.
+<2> Activator should always be scaled to a minimum of two instances to avoid downtime.
```

Comment:
It's fine to write the number out, but for consistency it's better to use the same format across the board
```suggestion
<2> Activator should always be scaled to a minimum of 2 instances to avoid downtime.
```